apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -4790,7 +4790,7 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
                     if (taxTypeArr.indexOf(ta.taxType) > -1) {
                         salesEntryClass.isOtherTaxApplicable = true;
                         let otherTaxModal = new SalesOtherTaxesModal();
-                        otherTaxModal.appliedOtherTax = { name: ta.name, uniqueName: ta.uniqueName };
+                        otherTaxModal.appliedOtherTax = { name: ta?.name, uniqueName: ta.uniqueName };
```

Comment:
you added ? for name and not for uniqueName. why? and how ta can be null of undefined here? Explain in detail